cmd/release/cmd/update.go

```diff
@@ -3,7 +3,10 @@ package cmd
 import (
 	"context"
 	"errors"
+	"fmt"
+	"log"
```

Comment:
Let's remove this in favor of regular CLI outut. We don't have anything consuming these logs and they're created at the time of execution so their timestamp values don't give us any extra value.

Reply:
Implemented